src/Components/CreateImageWizard/steps/Users/index.tsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+
+import { Form, Text, Title } from '@patternfly/react-core';
+
+import Empty from './component/Empty';
+
+const UsersStep = () => {
+  return (
+    <Form>
+      <Title headingLevel="h1" size="xl">
+        Users
+      </Title>
+      <Text>Add a user to your image.</Text>
+      <Empty />
```

Comment:
This looks great. Just a hint for your next PR, you'll want to use conditional rendering to display the empty state only if there is no user.
```javascript
user ? <UserInfo /> : <UserEmptyState />
```

Reply:
will do it in the next task thanks